src/GitVersionCore/SemanticVersionFormatValues.cs

```diff
@@ -109,7 +109,7 @@ public string Sha
 
         public string CommitDate
         {
-            get { return _semver.BuildMetaData.CommitDate.UtcDateTime.ToString("yyyy-MM-dd"); }
+            get { return _semver.BuildMetaData.CommitDate.UtcDateTime.ToString(_config.CommitDateFormat); }
```

Comment:
Is it perhaps a good idea to pass in `CultureInfo.InvariantCulture` here, just to avoid potential i18n problems or is it a "feature" that the date can be formatted in the locale of the process GitVersion is run under?